<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> publications | Mick C. J. Lee </title> <meta name="author" content="Mick C. J. Lee"> <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design. "> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://cjmlee.github.io/publications/"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?0afe9f0ae161375728f7bcc5eb5b4ab4"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">Mick </span> C. J.  Lee </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">Repositories </a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fa-solid fa-moon"></i> <i class="fa-solid fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description"></p> </header> <article> <p>For full publication list see <a href="https://scholar.google.com/citations?user=lp6yhQsAAAAJ&amp;hl=en" rel="external nofollow noopener" target="_blank">google scholar</a></p> <p>*equal Contributions<br> ^Co-Corresponding —</p> <div class="publications"> <h2 class="bibliography">2024</h2> <ol class="bibliography"><li> <div class="row"> <div id="RN8" class="col-sm-10"> <div class="title">Computationally defined and in vitro validated putative genomic safe harbour loci for transgene expression in human cells</div> <div class="author"> M. I. Autio ,  E. Motakis ,  A. Perrin ,  T. Bin Amin ,  Z. Tiang ,  D. V. Do ,  J. Wang ,  J. Tan ,  S. S. L. Ding ,  W. X. Tan ,  <u><b>C. J. M. Lee</b></u> ,  A. K. K. Teo ,  and  R. S. Y. Foo </div> <div class="periodical"> 2024 </div> <div class="periodical"> Autio, Matias I Motakis, Efthymios Perrin, Arnaud Bin Amin, Talal Tiang, Zenia Do, Dang Vinh Wang, Jiaxu Tan, Joanna Ding, Shirley Suet Lee Tan, Wei Xuan Lee, Chang Jie Mick Teo, Adrian Kee Keong Foo, Roger S Y eng 1610851033/Biomedical Research Council/ 202D8020/Agency for Science, Technology and Research/ England 2024/01/02 Elife. 2024 Jan 2;13:e79592. doi: 10.7554/eLife.79592. </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Selection of the target site is an inherent question for any project aiming for directed transgene integration. Genomic safe harbour (GSH) loci have been proposed as safe sites in the human genome for transgene integration. Although several sites have been characterised for transgene integration in the literature, most of these do not meet criteria set out for a GSH and the limited set that do have not been characterised extensively. Here, we conducted a computational analysis using publicly available data to identify 25 unique putative GSH loci that reside in active chromosomal compartments. We validated stable transgene expression and minimal disruption of the native transcriptome in three GSH sites in vitro using human embryonic stem cells (hESCs) and their differentiated progeny. Furthermore, for easy targeted transgene expression, we have engineered constitutive landing pad expression constructs into the three validated GSH in hESCs.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"><li> <div class="row"> <div id="RN7" class="col-sm-10"> <div class="title">Systematic in vivo candidate evaluation uncovers therapeutic targets for LMNA dilated cardiomyopathy and risk of Lamin A toxicity</div> <div class="author"> C. Y. Tan ,  P. S. Chan ,  H. Tan ,  S. W. Tan ,  <u><b>C. J. M. Lee</b></u> ,  J. W. Wang ,  S. Ye ,  H. Werner ,  Y. J. Loh ,  Y. L. Lee ,  M. Ackers-Johnson ,  R. S. Y. Foo ,  and  J. Jiang </div> <div class="periodical"> 2023 </div> <div class="periodical"> Tan, Chia Yee Chan, Pui Shi Tan, Hansen Tan, Sung Wei Lee, Chang Jie Mick Wang, Jiong-Wei Ye, Shu Werner, Hendrikje Loh, Ying Jie Lee, Yin Loon Ackers-Johnson, Matthew Foo, Roger S Y Jiang, Jianming eng Research Support, Non-U.S. Gov’t England 2023/10/16 J Transl Med. 2023 Oct 16;21(1):690. doi: 10.1186/s12967-023-04542-4. </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>BACKGROUND: Dilated cardiomyopathy (DCM) is a severe, non-ischemic heart disease which ultimately results in heart failure (HF). Decades of research on DCM have revealed diverse aetiologies. Among them, familial DCM is the major form of DCM, with pathogenic variants in LMNA being the second most common form of autosomal dominant DCM. LMNA DCM is a multifactorial and complex disease with no specific treatment thus far. Many studies have demonstrated that perturbing candidates related to various dysregulated pathways ameliorate LMNA DCM. However, it is unknown whether these candidates could serve as potential therapeutic targets especially in long term efficacy. METHODS: We evaluated 14 potential candidates including Lmna gene products (Lamin A and Lamin C), key signaling pathways (Tgfbeta/Smad, mTor and Fgf/Mapk), calcium handling, proliferation regulators and modifiers of LINC complex function in a cardiac specific Lmna DCM model. Positive candidates for improved cardiac function were further assessed by survival analysis. Suppressive roles and mechanisms of these candidates in ameliorating Lmna DCM were dissected by comparing marker gene expression, Tgfbeta signaling pathway activation, fibrosis, inflammation, proliferation and DNA damage. Furthermore, transcriptome profiling compared the differences between Lamin A and Lamin C treatment. RESULTS: Cardiac function was restored by several positive candidates (Smad3, Yy1, Bmp7, Ctgf, aYAP1, Sun1, Lamin A, and Lamin C), which significantly correlated with suppression of HF/fibrosis marker expression and cardiac fibrosis in Lmna DCM. Lamin C or Sun1 shRNA administration achieved consistent, prolonged survival which highly correlated with reduced heart inflammation and DNA damage. Importantly, Lamin A treatment improved but could not reproduce long term survival, and Lamin A administration to healthy hearts itself induced DCM. Mechanistically, we identified this lapse as caused by a dose-dependent toxicity of Lamin A, which was independent from its maturation. CONCLUSIONS: In vivo candidate evaluation revealed that supplementation of Lamin C or knockdown of Sun1 significantly suppressed Lmna DCM and achieve prolonged survival. Conversely, Lamin A supplementation did not rescue long term survival and may impart detrimental cardiotoxicity risk. This study highlights a potential of advancing Lamin C and Sun1 as therapeutic targets for the treatment of LMNA DCM.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"> <li> <div class="row"> <div id="RN4" class="col-sm-10"> <div class="title">Genomic enhancers in cardiac development and disease</div> <div class="author"> C. G. Anene-Nzelu ,  <u><b>C. J. M. Lee</b></u> ,  W. L. W. Tan ,  A. Dashi ,  and  R. S. Y. Foo </div> <div class="periodical"> 2022 </div> <div class="periodical"> Anene-Nzelu, Chukwuemeka G Lee, Mick C J Tan, Wilson L W Dashi, Albert Foo, Roger S Y eng Review England 2021/08/13 Nat Rev Cardiol. 2022 Jan;19(1):7-25. doi: 10.1038/s41569-021-00597-2. Epub 2021 Aug 11. </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The Human Genome Project marked a major milestone in the scientific community as it unravelled the  3 billion bases that are central to crucial aspects of human life. Despite this achievement, it only scratched the surface of understanding how each nucleotide matters, both individually and as part of a larger unit. Beyond the coding genome, which comprises only  2% of the whole genome, scientists have realized that large portions of the genome, not known to code for any protein, were crucial for regulating the coding genes. These large portions of the genome comprise the ’non-coding genome’. The history of gene regulation mediated by proteins that bind to the regulatory non-coding genome dates back many decades to the 1960s. However, the original definition of ’enhancers’ was first used in the early 1980s. In this Review, we summarize benchmark studies that have mapped the role of cardiac enhancers in disease and development. We highlight instances in which enhancer-localized genetic variants explain the missing link to cardiac pathogenesis. Finally, we inspire readers to consider the next phase of exploring enhancer-based gene therapy for cardiovascular disease.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="RN5" class="col-sm-10"> <div class="title">The genetic interactions between non-alcoholic fatty liver disease and cardiovascular diseases</div> <div class="author"> N. W. S. Chew ,  B. Chong ,  C. H. Ng ,  G. Kong ,  Y. H. Chin ,  W. Xiao ,  <u><b>C. J. M. Lee</b></u> ,  Y. Y. Dan ,  M. D. Muthiah ,  and  R. Foo </div> <div class="periodical"> 2022 </div> <div class="periodical"> Chew, Nicholas W S Chong, Bryan Ng, Cheng Han Kong, Gwyneth Chin, Yip Han Xiao, Wang Lee, Mick Dan, Yock Young Muthiah, Mark D Foo, Roger eng Review Switzerland 2022/08/30 Front Genet. 2022 Aug 10;13:971484. doi: 10.3389/fgene.2022.971484. eCollection 2022. </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The ongoing debate on whether non-alcoholic fatty liver disease (NAFLD) is an active contributor or an innocent bystander in the development of cardiovascular disease (CVD) has sparked interests in understanding the common mediators between the two biologically distinct entities. This comprehensive review identifies and curates genetic studies of NAFLD overlapping with CVD, and describes the colinear as well as opposing correlations between genetic associations for the two diseases. Here, CVD described in relation to NAFLD are coronary artery disease, cardiomyopathy and atrial fibrillation. Unique findings of this review included certain NAFLD susceptibility genes that possessed cardioprotective properties. Moreover, the complex interactions of genetic and environmental risk factors shed light on the disparity in genetic influence on NAFLD and its incident CVD. This serves to unravel NAFLD-mediated pathways in order to reduce CVD events, and helps identify targeted treatment strategies, develop polygenic risk scores to improve risk prediction and personalise disease prevention.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="RN6" class="col-sm-10"> <div class="title">BNIP-2 Activation of Cellular Contractility Inactivates YAP for H9c2 Cardiomyoblast Differentiation</div> <div class="author"> D. C. P. Wong ,  J. Xiao ,  T. W. Chew ,  M. Pan ,  <u><b>C. J. M. Lee</b></u> ,  J. W. Ang ,  I. Yow ,  T. Thivakar ,  M. Ackers-Johnson ,  N. J. W. Lee ,  R. S. Foo ,  P. Kanchanawong ,  and  B. C. Low </div> <div class="periodical"> 2022 </div> <div class="periodical"> Wong, Darren Chen Pei Xiao, Jingwei Chew, Ti Weng Pan, Meng Lee, Chang Jie Mick Ang, Jing Wen Yow, Ivan Thivakar, T Ackers-Johnson, Matthew Lee, Nicole Jia Wen Foo, Roger Sik-Yin Kanchanawong, Pakorn Low, Boon Chuan eng Mechanobiology Institute Singapore (MBI)/ National Research Foundation and the Ministry of Education Singapore/ MOE2016-T3-1-002/Ministry of Education (Singapore) Academic Research Fund Tier 3/ MOE2019-T2-2-014/Ministry of Education (Singapore) Academic Research Fund Tier2/ Research Support, Non-U.S. Gov’t Germany 2022/08/18 Adv Sci (Weinh). 2022 Nov;9(31):e2202834. doi: 10.1002/advs.202202834. Epub 2022 Aug 17. </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Rho GTPases and Hippo kinases are key regulators of cardiomyoblast differentiation. However, how these signaling axes are coordinated spatiotemporally remains unclear. Here, the central and multifaceted roles of the BCH domain containing protein, BNIP-2, in orchestrating the expression of two key cardiac genes (cardiac troponin T [cTnT] and cardiac myosin light chain [Myl2]) in H9c2 and human embryonic stem cell-derived cardiomyocytes are delineated. This study shows that BNIP-2 mRNA and protein expression increase with the onset of cTnT and Myl2 and promote the alignment of H9c2 cardiomyocytes. Mechanistically, BNIP-2 is required for the inactivation of YAP through YAP phosphorylation and its cytosolic retention. Turbo-ID proximity labeling corroborated by super-resolution analyses and biochemical pulldown data reveals a scaffolding role of BNIP-2 for LATS1 to phosphorylate and inactivate YAP in a process that requires BNIP-2 activation of cellular contractility. The findings identify BNIP-2 as a pivotal signaling scaffold that spatiotemporally integrates RhoA/Myosin II and LATS1/YAP mechanotransduction signaling to drive cardiomyoblast differentiation, by switching the genetic programming from YAP-dependent growth to YAP-silenced differentiation. These findings offer insights into the importance of scaffolding proteins in bridging the gap between mechanical and biochemical signals in cell growth and differentiation and the prospects in translational applications.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Lee2022.11.04.515249" class="col-sm-10"> <div class="title">CTCF mediates the Activity-by-contact derived cis-regulatory hubs</div> <div class="author"> <u><b>C. J. M. Lee</b></u> ,  Loı̈c Mangnier ,  Cory C. Padilla ,  Dominic Paul Lee ,  Wilson Tan ,  Peter Yiqing Li ,  Wen Hao Zheng ,  Louis Gan ,  Ching Kit Chen ,  Yee Phong Lim ,  Rina Miao Qin Wang ,  Steve Bilodeau ,  Alexandre Bureau ,  Roger Sik-Yin Foo^ ,  and  Chukwuemeka George Anene-Nzelu^ </div> <div class="periodical"> <em>bioRxiv</em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The 3D chromatin architecture establishes a complex network of genes and regulatory elements necessary for transcriptomic regulation in development and disease. This network can be modeled by cis-regulatory hubs (CRH) which underscore the local functional interactions between enhancers and promoter regions and differ from other higher order chromatin structures such as topological associated domains (TAD). The Activity-by-contact (ABC) model of enhancer-promoter regulation has been recently used in the identification of these CRHs, but little is known about the role of CTCF on the ABC scores and the consequent impact on CRHs. Here we show that the loss of CTCF leads to a reorganization of the enhancer-promoter interactions resulting in a re-distribution of ABC scores of the putative enhancers in C57/bl6 mouse hearts. The loss of CTCF also leads to a global reduction of the total number of CRHs, and an increase in the size of the CRHs due to increase in the number of elements within each hub. In addition, CTCF loss led to more CRHs that cross TAD boundaries. These results provide another layer of evidence to support the importance of CTCF in the formation of regulatory networks necessary for gene regulation.Competing Interest StatementThe authors have declared no competing interest.ChIP-seqChromatin immunoprecipitation with sequencingATAC-seqAssay for Transposase Accessible Chromatin with sequencingHi-CHigh throughput chromatin conformation capture with sequencingABCActivity by contact algorithm</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2020</h2> <ol class="bibliography"> <li> <div class="row"> <div id="RN1" class="col-sm-10"> <div class="title">Assigning Distal Genomic Enhancers to Cardiac Disease-Causing Genes</div> <div class="author"> *C. G. Anene-Nzelu ,  *W. L. W. Tan ,  <u><b>*C. J. M. Lee</b></u> ,  Z. Wenhao ,  A. Perrin ,  A. Dashi ,  Z. Tiang ,  M. I. Autio ,  B. Lim ,  E. Wong ,  H. S. Tan ,  B. Pan ,  M. P. Morley ,  K. B. Margulies ,  T. P. Cappola ,  and  R. S. Foo </div> <div class="periodical"> 2020 </div> <div class="periodical"> 1524-4539 R01 HL105993/HL/NHLBI NIH HHS/United States Letter United States 2020/09/01 Circulation. 2020 Sep;142(9):910-912. doi: 10.1161/CIRCULATIONAHA.120.046040. Epub 2020 Aug 31. </div> <div class="links"> <a href="https://www.ahajournals.org/doi/10.1161/CIRCULATIONAHA.120.046040" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="https://doi.org/10.1161/CIRCULATIONAHA.120.046040" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div id="RN3" class="col-sm-10"> <div class="title">Epigenomes of Human Hearts Reveal New Genetic Variants Relevant for Cardiac Disease and Phenotype</div> <div class="author"> W. L. W. Tan ,  C. G. Anene-Nzelu ,  E. Wong ,  <u><b>C. J. M. Lee</b></u> ,  H. S. Tan ,  S. J. Tang ,  A. Perrin ,  K. X. Wu ,  W. Zheng ,  R. J. Ashburn ,  B. Pan ,  M. Y. Lee ,  M. I. Autio ,  M. P. Morley ,  W. L. Tam ,  C. Cheung ,  K. B. Margulies ,  L. Chen ,  T. P. Cappola ,  M. Loh ,  J. Chambers ,  S. Prabhakar ,  R. S. Y. Foo ,  and  Charge-EchoGen Consortium Charge-Heart Failure Working Group </div> <div class="periodical"> 2020 </div> <div class="periodical"> Tan, Wilson Lek Wen Anene-Nzelu, Chukwuemeka George Wong, Eleanor Lee, Chang Jie Mick Tan, Hui San Tang, Sze Jing Perrin, Arnaud Wu, Kan Xing Zheng, Wenhao Ashburn, Robert John Pan, Bangfen Lee, May Yin Autio, Matias Ilmari Morley, Michael P Tam, Wai Leong Cheung, Christine Margulies, Kenneth B Chen, Leilei Cappola, Thomas P Loh, Marie Chambers, John Prabhakar, Shyam Foo, Roger S Y eng R01 HL105993/HL/NHLBI NIH HHS/ Research Support, N.I.H., Extramural Research Support, Non-U.S. Gov’t 2020/06/13 Circ Res. 2020 Aug 28;127(6):761-777. doi: 10.1161/CIRCRESAHA.120.317254. Epub 2020 Jun 12. </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>RATIONALE: Identifying genetic markers for heterogeneous complex diseases such as heart failure is challenging and requires prohibitively large cohort sizes in genome-wide association studies to meet the stringent threshold of genome-wide statistical significance. On the other hand, chromatin quantitative trait loci, elucidated by direct epigenetic profiling of specific human tissues, may contribute toward prioritizing subthreshold variants for disease association. OBJECTIVE: Here, we captured noncoding genetic variants by performing epigenetic profiling for enhancer H3K27ac chromatin immunoprecipitation followed by sequencing in 70 human control and end-stage failing hearts. METHODS AND RESULTS: We have mapped a comprehensive catalog of 47 321 putative human heart enhancers and promoters. Three thousand eight hundred ninety-seven differential acetylation peaks (FDR [false discovery rate], 5%) pointed to pathways altered in heart failure. To identify cardiac histone acetylation quantitative trait loci (haQTLs), we regressed out confounding factors including heart failure disease status and used the G-SCI (Genotype-independent Signal Correlation and Imbalance) test(1) to call out 1680 haQTLs (FDR, 10%). RNA sequencing performed on the same heart samples proved a subset of haQTLs to have significant association also to gene expression (expression quantitative trait loci), either in cis (180) or through long-range interactions (81), identified by Hi-C (high-throughput chromatin conformation assay) and HiChIP (high-throughput protein centric chromatin) performed on a subset of hearts. Furthermore, a concordant relationship between the gain or disruption of TF (transcription factor)-binding motifs, inferred from alternative alleles at the haQTLs, implied a surprising direct association between these specific TF and local histone acetylation in human hearts. Finally, 62 unique loci were identified by colocalization of haQTLs with the subthreshold loci of heart-related genome-wide association studies datasets. CONCLUSIONS: Disease and phenotype association for 62 unique loci are now implicated. These loci may indeed mediate their effect through modification of enhancer H3K27 acetylation enrichment and their corresponding gene expression differences (bioRxiv: https://doi.org/10.1101/536763). Graphical Abstract: A graphical abstract is available for this article.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2019</h2> <ol class="bibliography"><li> <div class="row"> <div id="RN2" class="col-sm-10"> <div class="title">Robust CTCF-Based Chromatin Architecture Underpins Epigenetic Changes in the Heart Failure Stress-Gene Response</div> <div class="author"> D. P. Lee ,  W. L. W. Tan ,  C. G. Anene-Nzelu ,  <u><b>C. J. M. Lee</b></u> ,  P. Y. Li ,  T. D. A. Luu ,  C. X. Chan ,  Z. Tiang ,  S. L. Ng ,  X. Huang ,  M. Efthymios ,  M. I. Autio ,  J. Jiang ,  M. J. Fullwood ,  S. Prabhakar ,  E. Lieberman Aiden ,  and  R. S. Foo </div> <div class="periodical"> 2019 </div> <div class="periodical"> Lee, Dominic Paul Tan, Wilson Lek Wen Anene-Nzelu, Chukwuemeka George Lee, Chang Jie Mick Li, Peter Yiqing Luu, Tuan Danh Anh Chan, Cheryl Xueli Tiang, Zenia Ng, Shi Ling Huang, Xingfan Efthymios, Motakis Autio, Matias I Jiang, Jianming Fullwood, Melissa Jane Prabhakar, Shyam Lieberman Aiden, Erez Foo, Roger Sik-Yin eng Research Support, Non-U.S. Gov’t 2019/02/06 Circulation. 2019 Apr 16;139(16):1937-1956. doi: 10.1161/CIRCULATIONAHA.118.036726. </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>BACKGROUND: The human genome folds in 3 dimensions to form thousands of chromatin loops inside the nucleus, encasing genes and cis-regulatory elements for accurate gene expression control. Physical tethers of loops are anchored by the DNA-binding protein CTCF and the cohesin ring complex. Because heart failure is characterized by hallmark gene expression changes, it was recently reported that substantial CTCF-related chromatin reorganization underpins the myocardial stress-gene response, paralleled by chromatin domain boundary changes observed in CTCF knockout. METHODS: We undertook an independent and orthogonal analysis of chromatin organization with mouse pressure-overload model of myocardial stress (transverse aortic constriction) and cardiomyocyte-specific knockout of Ctcf. We also downloaded published data sets of similar cardiac mouse models and subjected them to independent reanalysis. RESULTS: We found that the cardiomyocyte chromatin architecture remains broadly stable in transverse aortic constriction hearts, whereas Ctcf knockout resulted in approximately 99% abolition of global chromatin loops. Disease gene expression changes correlated instead with differential histone H3K27-acetylation enrichment at their respective proximal and distal interacting genomic enhancers confined within these static chromatin structures. Moreover, coregulated genes were mapped out as interconnected gene sets on the basis of their multigene 3D interactions. CONCLUSIONS: This work reveals a more stable genome-wide chromatin framework than previously described. Myocardial stress-gene transcription responds instead through H3K27-acetylation enhancer enrichment dynamics and gene networks of coregulation. Robust and intact CTCF looping is required for the induction of a rapid and accurate stress response.</p> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2024 Mick C. J. Lee. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2930004b8d7fcd0a8e00fdcfc8fc9f24"></script> <script defer src="/assets/js/common.js?4a129fbf39254905f505c7246e641eaf"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>